### PR TITLE
Lambdas

### PR DESCRIPTION
**Pull request by pentreathm**
Adding Lambdas endpoints API Specs to the definition.

### CODE REVIEW

File: openapi/paths/lambdas/lambdas@contentv2@scenes.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+get:
+  operationId: getScenes
+  summary: Get Scenes
+  tags:
+    - Lambdas
+  description: Retrieve the list of scenes in a given area
+  parameters:
```
Comment (agusaldasoro) [inline openapi/paths/lambdas/lambdas@contentv2@scenes.yaml:7]: All of this parameters are required

Comment (agusaldasoro) [reply openapi/paths/lambdas/lambdas@contentv2@scenes.yaml:7]: Also, the max size of the rectangle is 100x100

File: openapi/openapi.yaml [hunk 2]
```diff
@@ -58,4 +60,23 @@ paths:
     $ref: 'paths/lambdas/lambdas@collections@wearables.yaml'
   /lambdas/collections/wearables-by-owner/{address}:
     $ref: 'paths/lambdas/lambdas@collections@wearables-by-owner@{address}.yaml'
+  /lambdas/contentv2/scenes:
+    $ref: 'paths/lambdas/lambdas@contentv2@scenes.yaml'   
+  /lambdas/contentv2/parcel_info:
```
Comment (agusaldasoro) [inline openapi/openapi.yaml:65]: Personal opinion: This shouldn't be another endpoint, the `cids` should be another query parameter in `/lambdas/contentv2/scenes`. Anyway, the documentation for it is okay, changing that would mean a refactor of lambdas code.

File: openapi/openapi.yaml [hunk 3]
```diff
@@ -58,4 +60,23 @@ paths:
     $ref: 'paths/lambdas/lambdas@collections@wearables.yaml'
   /lambdas/collections/wearables-by-owner/{address}:
     $ref: 'paths/lambdas/lambdas@collections@wearables-by-owner@{address}.yaml'
+  /lambdas/contentv2/scenes:
+    $ref: 'paths/lambdas/lambdas@contentv2@scenes.yaml'   
+  /lambdas/contentv2/parcel_info: 
+    $ref: 'paths/lambdas/lambdas@contentv2@parcel_info.yaml'
```
Comment (agusaldasoro) [inline openapi/openapi.yaml:66]: Did you forget to add the route `/lambdas/contentv2/contents/:cid`?

File: openapi/paths/lambdas/lambdas@crypto@validate-signature.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
+post:
+  operationId: validateSignature
+  summary: Validated signed message
```
Comment (agusaldasoro) [inline openapi/paths/lambdas/lambdas@crypto@validate-signature.yaml:3]:
```suggestion
  summary: Validate signed message 
```

File: openapi/paths/lambdas/lambdas@crypto@validate-signature.yaml [hunk 5]
```diff
@@ -0,0 +1,31 @@
+post:
+  operationId: validateSignature
+  summary: Validated signed message 
+  tags:
+    - Lambdas
+  description: >- 
+    Given a signed message and it's authchain, validate it's authenticity.  
+    A message can be signed, for example, to prove ownership of the Entity pointers that they want to modify.
+  requestBody:
+    description: >-
```
Comment (agusaldasoro) [inline openapi/paths/lambdas/lambdas@crypto@validate-signature.yaml:10]: Can we make the body parameters have the same format as request body parameters have in `/content/denylist/{type}/{id}`?

File: openapi/paths/lambdas/lambdas@crypto@validate-signature.yaml [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+       <br/>- The last element of the chain contains the EHash and a signature
+       <br/>- In between are ephemeral keys that can be used for signing      
+    content:
+      application/json; charset=utf-8:
+        schema:
+          type: object        
+        example:
+          $ref: ../../components/examples/lambdas/body-validate-signature.json
+  responses:
+    '200':
```
Comment (agusaldasoro) [inline openapi/paths/lambdas/lambdas@crypto@validate-signature.yaml:23]: Can we document the response status code when the signature key is invalid?

File: openapi/components/examples/lambdas/200-realms.json [hunk 7]
```diff
@@ -0,0 +1,57 @@
+[
+    {
+        "serverName": "loki",
+        "url": "https://interconnected.online",
+        "layer": "amber",
```
Comment (agusaldasoro) [inline openapi/components/examples/lambdas/200-realms.json:5]: We should remove layers from this response, as they won't be present any more

File: openapi/paths/lambdas/lambdas@explore@realms.yaml [hunk 8]
```diff
@@ -0,0 +1,17 @@
+get:
+  operationId: getRealms
+  summary: Get Realms state
```
Comment (agusaldasoro) [inline openapi/paths/lambdas/lambdas@explore@realms.yaml:3]:
```suggestion
  summary: Get Realms status
```

File: openapi/openapi.yaml [hunk 9]
```diff
@@ -58,4 +60,23 @@ paths:
...
+  /lambdas/contracts/denylisted-names:
+    $ref: 'paths/lambdas/lambdas@contracts@denylisted-names.yaml'
+  /lambdas/crypto/validate-signature: 
+    $ref: 'paths/lambdas/lambdas@crypto@validate-signature.yaml'
+  /lambdas/explore/hot-scenes:
+    $ref: 'paths/lambdas/lambdas@explore@hot-scenes.yaml'
+  /lambdas/explore/realms:
+    $ref: 'paths/lambdas/lambdas@explore@realms.yaml'
+  /lambdas/profiles/{id}:
+    $ref: 'paths/lambdas/lambdas@profiles@{id}.yaml'
```
Comment (agusaldasoro) [inline openapi/openapi.yaml:80]: Please, also document `/lambdas/profiles?id=<PROFILE_ID_1>,<PROFILE_ID_2>` endpoint

File: openapi/paths/lambdas/lambdas@contentv2@scenes.yaml (hunk 1 above)
Comment (pentreathm) [reply openapi/paths/lambdas/lambdas@contentv2@scenes.yaml:9]: added

File: openapi/openapi.yaml (hunk 3 above)
Comment (pentreathm) [reply openapi/openapi.yaml:66]: it was missing, just added the endpoint definition

File: openapi/paths/lambdas/lambdas@crypto@validate-signature.yaml (hunk 6 above)
Comment (pentreathm) [reply openapi/paths/lambdas/lambdas@crypto@validate-signature.yaml:23]: Added

File: openapi/components/examples/lambdas/200-realms.json (hunk 7 above)
Comment (pentreathm) [reply openapi/components/examples/lambdas/200-realms.json:5]: done

File: openapi/openapi.yaml (hunk 9 above)
Comment (pentreathm) [reply openapi/openapi.yaml:118]: added

**Pull request merged**